frontend/src/components/sessions/TestSessionListItem.tsx

```diff
@@ -55,77 +55,94 @@ const TestSessionListItem = ({
   )} ${accessCode.slice(ACCESS_CODE_GROUP_SIZE)}`;
 
   return (
-    <HStack gap={2} pb={8} w="100%">
-      <Tooltip
-        bg="blue.300"
-        borderRadius={4}
-        hasArrow
-        label={classroomName}
-        p={2}
-        placement="left"
+    <Tooltip
+      bg="blue.300"
+      borderRadius={4}
+      hasArrow
+      isDisabled={status !== "past"}
+      label="Click to view statistics of this assessment and each student's performance."
+      p={2}
+      placement="bottom"
+    >
+      <HStack
+        _hover={{ bg: status === "past" ? "grey.100" : undefined }}
+        as={status === "past" ? "button" : undefined}
+        borderRadius={16}
+        gap={2}
+        p={4}
+        w="100%"
       >
-        <Tag
-          bg="blue.50"
-          borderRadius="full"
-          overflow="hidden"
-          size="lg"
-          w={36}
+        <Tooltip
+          bg="blue.300"
+          borderRadius={4}
+          hasArrow
+          label={classroomName}
+          p={2}
+          placement="left"
         >
-          <TagLeftIcon aria-hidden="true" as={BookIcon} color="blue.300" />
-          <TagLabel>
-            <Text
-              color="blue.300"
-              overflow="hidden"
-              textOverflow="ellipsis"
-              textStyle="smallerParagraph"
-              whiteSpace="nowrap"
-            >
-              {classroomName}
-            </Text>
-          </TagLabel>
-        </Tag>
-      </Tooltip>
-      <VStack align="start">
-        <Text
-          color={status === "past" ? "grey.300" : "blue.300"}
-          textStyle="subtitle2"
-        >
-          {testName}
-        </Text>
-        <Text
-          color={status === "past" ? "grey.200" : "blue.200"}
-          textStyle="mobileParagraph"
-        >
-          {STATUS_LABELS[status]} {formatDate(targetDate)}
-        </Text>
-      </VStack>
-      {status !== "past" && (
-        <Copyable label="Access Code" value={formattedAccessCode} />
-      )}
-      <Spacer />
-      {stats && (
-        <>
-          <Text color="grey.300" textStyle="mobileParagraph">
-            Mean: {stats.mean}%
-          </Text>
-          <Text color="grey.300" textStyle="mobileParagraph">
-            Median: {stats.median}%
-          </Text>
-          <Text color="grey.300" textStyle="mobileParagraph">
-            Completion Rate: {stats.completionRate}%
+          <Tag
+            bg="blue.50"
+            borderRadius="full"
+            maxWidth={36}
```

Comment:
```suggestion
            maxWidth={40}
```
To match design spec of 161px width? Let me know if i'm converting that incorrectly

Reply:
Yes good suggestion! For the other points above, I addressed the first one. For the second one, I'll address this in the other PR since otherwise there will be merge conflicts. For the last one, I think it would be quite difficult to make separate components without duplicating most of the layout and functionallty. Perhaps turning some of these style props into additional map-objects rather than including ternary statements might help? Let me know what you think is best, but I'm in favour of avoiding a refactor unless the components really differ a lot functionally by status.

Reply:
okay sounds good. no refactor 👍 i'm happy with as is 😀